packages/eslint-plugin-baseui/bump.js

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// Sets local package.json to current baseui version.
+
+'use strict';
+
+const fs = require('fs');
+const baseuiVersion = require('../../package.json').version;
+const packageJSON = fs.readFileSync('package.json', 'utf-8');
```

Comment:
probably it's safer to also parse, and update the version field - while being unlikely, it can happen that another string will match `0.0.0` first

Reply:
should be safe since `replace` happens only for the first match

Reply:
yeah, that makes sense - probably I was overthinking it (what if we change the order of props in the package.json file)